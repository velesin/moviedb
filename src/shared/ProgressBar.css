.ProgressBar {
    margin-top: 64px;
    text-align: center;
    font-style: italic;
    font-size: 24px;
    color: #ffffff;
    font-weight: 200;
    letter-spacing: 8px;
}