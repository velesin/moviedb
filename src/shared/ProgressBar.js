import React from 'react';

import './ProgressBar.css';

export function ProgressBar() {
    return (
        <div className="ProgressBar">Loading...</div>
    );
}