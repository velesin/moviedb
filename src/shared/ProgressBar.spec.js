import React from 'react';
import { shallow } from 'enzyme';

import { ProgressBar } from './ProgressBar';

it('renders `Loading` message', () => {
    const progress = shallow(<ProgressBar />);
    expect(progress.getElement()).toMatchSnapshot();
});
