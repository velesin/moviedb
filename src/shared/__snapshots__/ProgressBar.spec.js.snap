// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders \`Loading\` message 1`] = `
<div
  className="ProgressBar"
>
  Loading...
</div>
`;
